cdap-ui/app/cdap/text/text-en.yaml

```diff
@@ -1678,6 +1678,30 @@ features:
       timeRangeTitle: Select time range
       to: To
 
+  Home:
+    Administrations:
+      description: Manage system and namespace level settings and preferences.
+      namespaceLabel: Namespace
+      systemLabel: System
+      title: Administrations
```

Comment:
Pretty sure it is more common to write without the 's' at the end -  I haven't seen it with an 's' before.  We do this in our existing UI as well. 

<img width="701" alt="Screen Shot 2019-09-09 at 12 09 31 PM" src="https://user-images.githubusercontent.com/16228294/64559263-b90ffb00-d2fa-11e9-8e50-eb0e6365f7c3.png">